x/launch/client/cli/tx_request_remove_account.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdRequestRemoveAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "request-remove-account [chainID] [address]",
+		Short: "Request to remove an account",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
```suggestion
		Args:  cobra.MinimumNArgs(2),
```

Reply:
I put one because I keep the second parameter as optional. If the second arg is not passed, we will use the message signer address. Do you think this makes sense?

https://github.com/tendermint/spn/pull/220/files#diff-c1d5d2c3cae4016c74cab175f476f90c088d4622ab88a749961967f2608a1fdfR22-R26